examples/Dockerfile

```diff
@@ -14,39 +14,40 @@
 # limitations under the License.
 #
 
-FROM maven:3.6.0-jdk-8-slim
+FROM eclipse-temurin:11-jre-jammy
```

Comment:
What is the reason for using eclipse-temurin instead of the maven image?

Reply:
We do not need maven anymore. I was looking for the smallest JDK11 image that is available for both arm64 and amd64 platforms